### DISCUSSION TITLE
Bump tags of ollama/ollama?

### DISCUSSION THREAD

**Discussion by anoordover** [CommunityToolkit/Aspire#292] [General]
Should we bump the tag of ollama/ollama to 0.4.5?
I can't find the tags of open-webui. Should we bump this tag as well?

**Comment by aaronpowell** [comment]
Yep, I'd seen that release drop today, I just hadn't gotten to creating the issue and PR.

As for OpenWebUI, that ships via the GitHub Packages container registry and it looks like 0.4.4 is their latest - https://github.com/open-webui/open-webui/releases/tag/v0.4.4

**Reply by aaronpowell** [reply]
This will ship in the next release
